pom.xml

```diff
@@ -44,28 +44,24 @@
         <url>https://github.com/predix/spring-metering-filter</url>
     </scm>
     <properties>
-        <spring-security-oauth2.version>2.0.3.RELEASE</spring-security-oauth2.version>
+        <spring-security-oauth2.version>2.5.0.RELEASE</spring-security-oauth2.version>
         <commons-lang.version>2.6</commons-lang.version>
```

Comment:
Change commons-lang to commons-lang3, there's a package change.
Also commons-collections to commons-collections4.

Reply:
@metacubed, the only method used from apache commons StringUtils is isNotEmpty().  I am thinking of changing it to spring framework StringUtils hasText().  One less package to maintain.  What do you think?

Reply:
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StringUtils.html#hasText-java.lang.String-

http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNotEmpty-java.lang.CharSequence-

Both return true if the String is not null, its length is greater than 0, and it contains at least one non-whitespace character.

Reply:
Sounds good, @irinaepshteyn.